openldap/osfingermap.yaml

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+#
+# Setup variables using grains['osfinger'] based logic.
+# You just need to add the key:values for an `osfinger` that differ
+# from `defaults.yaml` + `osarch.yaml` + `os_family.yaml` + `osmap.yaml`.
+# Only add an `osfinger` which is/will be supported by the formula.
+#
+# If you do not need to provide defaults via the `os_finger` grain,
+# you will need to provide at least an empty dict in this file, e.g.
+# osfingermap: {}
+---
+# os: Ubuntu
+Ubuntu-18.04: {}
+
+# os: CentOS
+CentOS-6: {}
+CentOS-7: {}
```

Comment:
Ouch, that's actually wrong in the `template-formula` itself.  It's actually `CentOS Linux-7` (`CentOS-6` is correct).

Reply:
I've added a fix for this in https://github.com/saltstack-formulas/template-formula/pull/184.